include/editormap.hpp

```diff
@@ -33,7 +37,7 @@ class EditorMap : public Dummy::Core::GraphicMap {
     void save();
     void resize(std::uint16_t, std::uint16_t);
 private:
-    void _saveBlockingLayer();
+    void _saveBlockingLayers();
```

Comment:
identifiers starting with `_` are reserved ones in C/C++

Reply:
What do you mean by that?

For now this is an inofficial convention on how to name private methods in the project.

What do you suggest otherwise?

Reply:
Every identifier starting with a `_` are reserved for the language as stated in https://stackoverflow.com/a/228797
Since the methods are private, no need to prefix them with a `_` (even if it's to know that it's a private method when you are coding, IDE are smart enough to tell you that)

Reply:
I’ll keep that in mind; plus I might start a refactoring session to abide by such rules. Tyvm!